participants/filecoin.net-disk.md

```diff
@@ -0,0 +1,56 @@
+# <Project Name>
+Filecoin.Net Disk
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+My project name is Filecoin.Net Disk
+My slack username is cannon, slack emil is cannon3758@outlook.com
+
+## (1) Brief Project Description
+This project is provide data store for users, which is use web UI interface to operation.
+And then we use lotus for data storage and retrieve deal from filecoin blockchain 
+
+
+## (2) Link to Project UI
+http://183.60.252.190:3334
+
+## (3) What does your application/UI do?
+Users can storage or retrieve data from filecoin network by this application 
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+My data is  sorts of video , we have the rights to the data for my plan. We have the data license available online. The data does'nt contain any  sensitive or illegal information 
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+We have probably 100T data planning  store to the Filecoin Network
+
+## (7) How are you structuring the data?
+Our data is manly for video format such as mp4, avi, and so on 
+
+## (8) What pre-processing are you doing before ingesting the data?
+
+Our size storage deal size is less the 32G 
+We are use hash cid to and filename to index the data in our application 
+
+## (9)  What tech stack will you use to store the data?
+We are use lotus
+
+## (10) How will you retrieve the data?
+We use lotus to retrieve data in our application from filecoin network.
+May be retrieve data in every time
+We retrieve data from filecoin network directly, dont't need to caching layer 
```

Comment:
In the UI, I don't see a button to retrieve the data. How does someone retrieve the data from your application?

Reply:
click 'copy hash' button,  paste into Text box， and then click download button